jax/_src/numpy/ufuncs.py

```diff
@@ -2555,10 +2555,15 @@ def _float_divmod(x1: ArrayLike, x2: ArrayLike) -> tuple[Array, Array]:
   mod = lax.rem(x1, x2)
   div = lax.div(lax.sub(x1, mod), x2)
 
-  ind = lax.bitwise_and(mod != 0, lax.sign(x2) != lax.sign(mod))
+  ind = lax.bitwise_and(mod != 0, signbit(x2) != signbit(mod))
```

Comment:
I think we could stick with `lax.sign` here, since we're looking at non-zero values (and it lowers more directly to efficient XLA ops).

Reply:
Done.